### PR TITLE
fix: change `plugin` option format

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
**changed plugin option format**
* as-is
`function Plugin(eventEmitter: Emitter, options: PluginOptions): PluginInfo`
* to-be
`function Plugin(context: PluginContext, options: PluginOptions): PluginInfo`
`PluginContext` includes `EventEmitter`, `i18n`, partial prosemirror state and view function.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jung-han — APPROVED** [review]
완료입니다!

File: apps/editor/types/editor.d.ts [hunk 1]
```diff
@@ -100,13 +100,25 @@ export interface EditorPluginInfo {
...
+  i18n: I18n;
   pmState: { Plugin: typeof Plugin; Selection: typeof Selection };
   pmView: { Decoration: typeof Decoration; DecorationSet: typeof DecorationSet };
 }
+
 export type PluginFn = (
-  eventEmitter: Emitter,
-  options: PluginDefaultOptions & any
+  context: PluginContext,
+  options?: Recored<string, any>
```
Comment (lja1018) [inline apps/editor/types/editor.d.ts:121]: `Record` 오타같습니다!

**Review by lja1018 — APPROVED** [review]
리뷰완료입니다.

**Review by jungeun-cho — APPROVED** [review]
이견없습니다~!

**Review by dotaitch — APPROVED** [review]
리뷰 완료합니다

**Pull request merged**